docs/CLIUsage.md

```diff
@@ -0,0 +1,262 @@
+        Name:  par
+        Steps:
+          Name:  run-step
+          Tasks:
+            run
+        Strategy:  serial
+      Strategy:    serial
+  Tasks:
+    Run:
+      Resources:
+        job.yaml
+  Templates:
+    Job . Yaml:  apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: default
+  name: {{PLAN_NAME}}-job
+spec:
+  template:
+    metadata:
+      name: {{PLAN_NAME}}-job
+    spec:
+      restartPolicy: OnFailure
+      containers:
+      - name: bb
+        image: busybox:latest
+        imagePullPolicy: IfNotPresent
+        command:
+        - /bin/sh
+        - -c
+        - "echo {{PLAN_NAME}} for v1 && echo Going to sleep for {{SLEEP}} seconds && sleep {{SLEEP}}"
+
+  Version:  1.0.0
+Events:     <none>
+```
+
+When looking at the particular `PlanExecution` we are able to see the status of the current applied plan.
+
+```bash
+$ kubectl describe planexecution up-deploy-493146000
+  Name:         up-deploy-493146000
+  Namespace:    default
+  Labels:       framework-version=upgrade-v1
+                instance=up
+  Annotations:  <none>
+  API Version:  maestro.k8s.io/v1alpha1
+  Kind:         PlanExecution
+  Metadata:
+    Cluster Name:        
+    Creation Timestamp:  2018-12-14T19:26:44Z
+    Generation:          1
+    Owner References:
+      API Version:           maestro.k8s.io/v1alpha1
+      Block Owner Deletion:  true
+      Controller:            true
+      Kind:                  Instance
+      Name:                  up
+      UID:                   3101bbe5-ffd6-11e8-abd5-080027d506c7
+    Resource Version:        63815
+    Self Link:               /apis/maestro.k8s.io/v1alpha1/namespaces/default/planexecutions/up-deploy-493146000
+    UID:                     31037dd0-ffd6-11e8-abd5-080027d506c7
+  Spec:
+    Instance:
+      Kind:       Instance
+      Name:       up
+      Namespace:  default
+    Plan Name:    deploy
+  Status:
+    Name:  deploy
+    Phases:
+      Name:   par
+      State:  COMPLETE
+      Steps:
+        Name:    run-step
+        State:   COMPLETE
+      Strategy:  serial
+    State:       COMPLETE
+    Strategy:    serial
+  Events:        <none>
+```
+
+The status information for the `Active-Plan` is nested in this part:
+
+```bash
+  Status:
+    Name:  deploy
+    Phases:
+      Name:   par
+      State:  COMPLETE
+      Steps:
+        Name:    run-step
+        State:   COMPLETE
+      Strategy:  serial
+    State:       COMPLETE
+    Strategy:    serial
+```
+
+Our tree view makes this information more readable to the user and creates a better user experience through one command rather than picking from multiple responses the bits and pieces.
+
+# Get the History to PlanExecutions
+
+This is helpful if you want to find out which plan ran on your instance to a particular `FrameworkVersion`.
+Let's say you want to know all plans that ran for the instance `up` and its FrameworkVersion `upgrade-v1`:
+
+```bash
+$ maestroctl plan history upgrade-v1 --instance=up
```

Comment:
I dont like optional things being command line parameters.  Maybe make the instance the command line argument and have an option flag that limits the history to the FV provided (or not)